.github/workflows/CD.yml

```diff
@@ -13,7 +13,7 @@
 
 name: C/D
 
-on:        
```

Comment:
Anyway you can leave out the formatting changes?

Reply:
so i've copied this from khanhs release and i think all its doing is removing white spaces which shouldnt be an issue